R/FilteredData.R

```diff
@@ -554,7 +548,7 @@ FilteredData <- R6::R6Class( # nolint
     #' @param active_datanames (`reactive`)
     #'   defining subset of `self$datanames()` to be displayed.
     #' @return `shiny.tag`
-    ui_active = function(id, active_datanames = self$datanames()) {
+    ui_active = function(id, active_datanames = self$datanames(), allow_add = TRUE) {
```

Comment:
`allow_add` is not an option for developer - it should be provided through `teal_slices(allow_add = FALSE)`. In `teal` we shouldn't use `allow_add` directly, it should be done automatically here in `teal.slice`

```suggestion
    ui_active = function(id, active_datanames = self$datanames()) {
```